lua/null-ls/builtins/code_actions/gitsigns.lua

```diff
@@ -27,7 +28,14 @@ return h.make_builtin({
                 table.insert(actions, {
                     title = name_to_title(name),
                     action = function()
-                        vim.api.nvim_buf_call(params.bufnr, action)
+                        local cb = action
+                        if range_actions[name] then
```

Comment:
```suggestion
                        if params.range and range_actions[name] then
```

This will only be defined if the original request has a range, so we should add a check.